english/virtualreality/voice/strangeVoice.md

```diff
@@ -0,0 +1,4 @@
+“You might have noticed that removing your Rift does not return you to reality.  This is working as intended.”
+
+As you inspect your head with your hands, you quickly realize that the hands you are seeing do not match up with the hands you can actually feel.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.